src/Date/CalendarEdit.tsx

```diff
@@ -165,11 +176,26 @@ function CalendarInputPure(
     const year = Number(date.slice(yearIndex, yearIndex + 4))
     const month = Number(date.slice(monthIndex, monthIndex + 2))
 
+    if (year % 400 === 0 || (year % 100 !== 0 && year % 4 === 0))
+      monthLength[1] = 29
+
     if (Number.isNaN(day) || Number.isNaN(year) || Number.isNaN(month)) {
       setError(inputFormat)
       return
     }
 
+    if (month === 0 || month > 12) {
+      setError('MM should be in range of 1-12')
```

Comment:
error's should be configurable and be added to examples + readme so users from other countries can configure the right errors